x-pack/legacy/plugins/ml/public/application/jobs/new_job/pages/new_job/page.tsx

```diff
@@ -153,17 +161,27 @@ export const Page: FC<PageProps> = ({ existingJobsAndGroups, jobType }) => {
     <Fragment>
       <EuiPage style={{ backgroundColor: 'inherit' }} data-test-subj={`mlPageJobWizard ${jobType}`}>
         <EuiPageBody>
-          <EuiPageContentBody>
-            <Wizard
-              jobCreator={jobCreator}
-              chartLoader={chartLoader}
-              resultsLoader={resultsLoader}
-              chartInterval={chartInterval}
-              jobValidator={jobValidator}
-              existingJobsAndGroups={existingJobsAndGroups}
-              firstWizardStep={firstWizardStep}
-            />
-          </EuiPageContentBody>
+          <EuiPageContent>
+            <EuiPageContentHeader>
+              <EuiPageContentHeaderSection>
+                <EuiTitle>
+                  <h2>Create job: {getJobCreatorTitle(jobCreator)} </h2>
```

Comment:
This text should be wrapped in a `<FormattedMessage />` element for translation.